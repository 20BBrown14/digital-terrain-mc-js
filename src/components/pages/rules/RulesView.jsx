import React from 'react';

/**
 * Rules page view.
 */
const RulesView = () => (
  <div>
    Rules Page
  </div>
);

export default RulesView;
