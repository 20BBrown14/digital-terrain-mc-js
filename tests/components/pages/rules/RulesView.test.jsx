import React from 'react';
import RulesView from '../../../../src/components/pages/rules/RulesView';

describe('RulesView', () => {
  it('renders a default view', () => {
    const view = render(<RulesView />);
    expect(view).toMatchSnapshot();
  });
});
