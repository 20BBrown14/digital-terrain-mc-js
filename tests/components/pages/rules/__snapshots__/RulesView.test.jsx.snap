// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RulesView renders a default view 1`] = `
<div>
  Rules Page
</div>
`;
